rai/node/node.cpp

```diff
@@ -3878,6 +3878,30 @@ void rai::active_transactions::announce_votes ()
 					auto tally_l (election_l->tally (transaction));
 					election_l->log_votes (tally_l);
 				}
+				/* Escalation for long unconfirmed elections
+				Start new elections for previous block & source
+				if there are less than 100 active elections */
+				if (i->announcements % announcement_long == 1 && roots.size () < 100)
```

Comment:
Is this supposed to be:
`if (i->announcements / announcement_long > 0 && i->announcements % announcement_long == 1 && roots.size () < 100)`

So that it doesn't fire when i->announcements is 1 ?

Reply:
It's inside of
`if (i->announcements > announcement_long)`

Reply:
Oops !